src/services/outliningElementsCollector.ts

```diff
@@ -16,6 +16,18 @@ namespace ts.OutliningElementsCollector {
             }
         }
 

```

Comment:
Can you move the body of `collectElements` above to precede local functions? It's not strictly-speaking necessary for this change, but makes reading `collectElements` easier and is consistent with defining local functions last elsewhere.